streamer/cloud_node.py

```diff
@@ -85,21 +85,19 @@ def check_access(bucket_url: str) -> None:
                             universal_newlines=True)
     # If the command failed, raise an error.
     if status.returncode != 0:
-      message = """Unable to write to cloud storage URL: {}
-
-Please double-check that the URL is correct, that you are signed into the
-Google Cloud SDK or Amazon AWS CLI, and that you have access to the
-destination bucket.
-
-Additional output from gsutil:
-  {}""".format(bucket_url, status.stderr)
+      message = ("Unable to write to cloud storage URL: {}\n\n"
```

Comment:
@TheModMaker
Would that be ok?, i am trying to keep the indentation because it causes visual problems in vscode and i think other text editors as well.